chart/templates/dags-persistent-volume-claim.yaml

```diff
@@ -19,7 +19,7 @@
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: {{ .Release.Name }}-dags
+  name: {{ template "airflow_dags_volume_claim" . }}
```

Comment:
Because:
https://github.com/apache/airflow/blob/5e13c372860a28256bf6e572bf7349f3dd6b8b0c/chart/templates/scheduler/scheduler-deployment.yaml#L203-L207